src/CONST.js

```diff
@@ -311,12 +311,34 @@ const CONST = {
         GSD: 'gsd',
         DEFAULT: 'default',
     },
+    JSON_CODE: {
+        SUCCESS: 200,
+        NOT_AUTHENTICATED: 407,
+        EXP_ERROR: 666,
+    },
     ERROR: {
+        XHR_FAILED: 'xhrFailed',
         API_OFFLINE: 'session.offlineMessageRetry',
         UNKNOWN_ERROR: 'Unknown error',
         REQUEST_CANCELLED: 'AbortError',
         FAILED_TO_FETCH: 'Failed to fetch',
         ENSURE_BUGBOT: 'ENSURE_BUGBOT',
+        NETWORK_REQUEST_FAILED: 'Network request failed',
+        SAFARI_DOCUMENT_LOAD_ABORTED: 'cancelled',
+        FIREFOX_DOCUMENT_LOAD_ABORTED: 'NetworkError when attempting to fetch resource.',
+        IOS_NETWORK_CONNECTION_LOST: 'The network connection was lost.',
```

Comment:
Is it possible to distinguish these `NETWORK_CONNECTION_LOST` errors by their status code or something other than the message? If Safari surfaces https errors with messages translated based on the device, it seems like doing error handling based on that message will be unreliable. We can't / won't try to catch the message in every language, so I wonder if we should just not include these Russian and Swedish error messages at all?

Overall you have more context about these errors than I do.

Reply:
I have no idea if there is any additional information available or if it's possible to identify these errors by anything other than the message string. I only know about these because we are logging them.